src/Symfony/Component/Process/ExecutableFinder.php

```diff
@@ -56,13 +56,20 @@ public function find($name, $default = null, array $extraDirs = array())
             $searchPath = explode(PATH_SEPARATOR, ini_get('open_basedir'));
             $dirs = array();
             foreach ($searchPath as $path) {
-                if (is_dir($path)) {
-                    $dirs[] = $path;
-                } else {
-                    if (basename($path) == $name && is_executable($path)) {
-                        return $path;
+                try {
+                    if (is_dir($path)) {
+                        $dirs[] = $path;
+                    } else {
+                        if (basename($path) == $name && is_executable($path)) {
+                            return $path;
+                        }
                     }
                 }
+                catch (\ErrorException $e) {
```

Comment:
@nicolas-grekas I replaced the try/catch with an `@`.

The relevant stack frame contains `HandleExceptions->handleError('2', 'is_dir(): open_basedir restriction in effect. File(/srv/www/example.com/web) is not within the allowed path(s) ...` where `HandleExceptions` is an instance of `Illuminate\Foundation\Bootstrap\HandleExceptions`.